src/confcom/azext_confcom/security_policy.py

```diff
@@ -470,6 +478,31 @@ def populate_policy_content_for_all_images(
                                 }
                             )
 
+                    if (deepdiff.DeepDiff(image.get_user(), config.DEFAULT_USER, ignore_order=True) == {}
+                            and image_info.get("User") != ""):
```

Comment:
would this be better as: `and image_info.get("User") in ["user", "user:group", "uid", "uid:gid", "user:gid", "uid:group"]` ?

Reply:
No because user, uid, group, and gid could be anything. The comment just shows the possible formatting of the user string

Reply:
gotcha, I would update the comment to say that. "valid values are in the form .... where each entry is either a number or a string" etc.